deploy_pulp3/deploy-pulp3.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  become: true
+  become_method: sudo
```

Comment:
Could do away with this line, since the default value for `become_method` is `sudo`.